src/components/Form/Accordion/Accordion.jsx

```diff
@@ -45,6 +45,17 @@ export const scrollToBottom = (selector) => {
   window.scroll({ top: el.offsetTop, left: 0, behavior: 'smooth' })
 }
 
+/**
+ * Checks if the validator is the expected type and contains an isValid func.
+ * TODO: Remove once everything has been refactored
+ */
+export const validValidator = (validator) => {
+  return validator &&
+    typeof validator === 'function' &&
+    !!validator.prototype.isValid &&
```

Comment:
Tagging @floresj on this.

We do plan (correct if I'm wrong @floresj) to remove this bit of code once all the sections have been moved over. We have the extra checks b/c this was being used in one way and is being changed so didn't want to break anything during the transition.

Reply:
Gotcha

Reply:
The typeof check ensures it's a function (function constructor) and the isValid check ensures that an isValid function exists within the instance of the function.

Reply:
Right.  My comment is on the wrong line.  Line 54 makes sense, but line 56 is another `typeof` check.  If `validator.prototype.isValid` does not exist, then `typeof validator.prototype.isValid` will return `undefined`.  The check on 56 covers both existence *and* type checking.

For reference, since Github won't expand the code block downward, here's the full function:

```js
export const validValidator = (validator) => {
  return validator &&
    typeof validator === 'function' &&
    !!validator.prototype.isValid &&
    typeof validator.prototype.isValid === 'function' // <-- line 56
}
```

If I'd put my comment on the correct line, I think it would have made more sense. 😬  There's functionally nothing wrong with it as-is, though.  I was just flagging it as a potential technical debt issue in the future.  Extra conditions increase the complexity (which is really low here 👍 ) and can throw a future developer for a loop.  I think in this case, though, it's clear what's happening (the `!!` always makes me pause, but I don't know a better way to coerce into a boolean reliably).

Reply:
@mgwalker oooo gotcha! Totally makes sense now 👍 👍